### ISSUE TITLE
Paddle里有类似tf layers.Masking这样的接口吗？

### ISSUE THREAD

**Issue by christine415** [PaddlePaddle/Paddle#31567]
Paddle里有类似tf layers.Masking这样的接口吗？
参考：https://www.tensorflow.org/api_docs/python/tf/keras/layers/Masking

**Comment by MrChengmo** [comment]
在Paddle 1.8.5及之前版本中，有sequence_mask op可以实现类似的功能， 链接 ： https://www.paddlepaddle.org.cn/documentation/docs/zh/1.8/api_cn/layers_cn/sequence_mask_cn.html#cn-api-fluid-layers-sequence-mask

**Comment by christine415** [comment]
> 在Paddle 1.8.5及之前版本中，有sequence_mask op可以实现类似的功能， 链接 ： https://www.paddlepaddle.org.cn/documentation/docs/zh/1.8/api_cn/layers_cn/sequence_mask_cn.html#cn-api-fluid-layers-sequence-mask

有使用的例子吗？
